src/plugins/janus_streaming.c

```diff
@@ -10192,6 +10257,13 @@ static void janus_streaming_relay_rtp_packet(gpointer data, gpointer user_data)
 					rtp.extensions.min_delay = s->min_delay;
 					rtp.extensions.max_delay = s->max_delay;
 				}
+				if(session->abscapturetime_ext && (session->abscapturetime_src_ext_id > 0 && session->abscapturetime_src_ext_id < 15)) {
```

Comment:
`session->abscapturetime_src_ext_id > 0` is enough as a check: the range of the ID is already sanitized when creating the mountpoints.

Reply:
fixed